<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectSettingsDialog</class>
 <widget class="QDialog" name="ObjectSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="50,45,5">
     <item>
      <widget class="QLabel" name="lObject">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabs">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bSave">
         <property name="text">
          <string>Save and Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bApply">
         <property name="toolTip">
          <string>Saves the settings of this object and all objects using the same animation.</string>
         </property>
         <property name="text">
          <string>Save Across All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
